sources/packages/web/src/components/aest/students/assessment/History.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-container>
+        <body-header title="History" class="m-1"> </body-header>
+        <span>A history of assessments</span>
```

Comment:
The body header also has a sub-title. Is it what we are looking for here?
![image](https://user-images.githubusercontent.com/61259237/160470295-7c6fd777-7cb5-4604-9027-23bca489a83a.png)